foursight_core/react/api/react_api.py

```diff
@@ -1110,6 +1122,25 @@ def reactapi_checks_registry(self, request: dict, env: str) -> Response:
         ignored(request, env)
         return self.create_success_response(self._checks.get_registry())
 
+    def reactapi_checks_validation(self, request: dict, env: str) -> Response:
+        """
+        Called from react_routes for endpoint: GET /{env}/checks_validation
+        Returns information about any problems with the checks setup.
+        """
+        ignored(request, env)
+        checks_actions_registry = self._checks.get_registry()
+        actions_with_no_assocated_check = [name for name in checks_actions_registry
```

Comment:
typo: `assocated` s.b. `associated`. Other references would need to be fixed.

Reply:
Thanks